app/src/main/java/com/lucascabral/marvelsuperheroes/presenter/view/MarvelYoutubeActivity.kt

```diff
@@ -40,6 +39,20 @@ class MarvelYoutubeActivity : AppCompatActivity() {
         return true
     }
 
+    private fun alertDialogEmptyList() {
```

Comment:
Uma sugestão é manter as funções que fazem algum tipo de ação com um verbo no início, por exemplo: `showAlertDialogEmptyList`. Fica mais fácil de entender o que a função vai executar.

Reply:
Feito.